src/draw/shapes.typ

```diff
@@ -1385,6 +1398,12 @@
 ///
 /// Elements hidden via @@hide() are ignored.
 ///
+/// = parameters
+///
+/// = Anchors
+///   / centroid: Centroid of the _closed and non self-intersecting_ shape. Only exists if `close` is true.
+///   Supports path anchors and shapes where all vertices share the same z-value.
+///
```

Comment:
same notes as before, remove parameters heading, and move the rest to the existing anchors subheading below